### ISSUE TITLE
Ability to select groups from adder

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#468]
From Zendesk ticket: https://hypothesis.zendesk.com/agent/tickets/2190
### Problem you are trying to address with this feature
This user separates annotations into two private groups ("Things I don't understand" and "Things to ponder"). They would like to be able to select a group from the adder so they can easily switch between groups while annotating a single page/document. 

This user's case could be helped by using tags rather than groups to separate out their annotations, but the user would not have an activity page for both of those groups.

Also, the ability to select a group could help clear up confusion for education users, who often find that students annotate in the "Public" layer rather than the designated group for the class.

**Comment by seanh** [comment]
@klemay I think "select group from adder" is a solution rather than a problem. What's the problem that we can solve for the user here? It seems to me that it's making it easier to annotate in group B, given that the group I'm currently in (selected in the client's groups menu) is group A not group B? That is, make it easier to post an annotation into a different group, other than the one currently selected in the sidebar.

**Comment by klemay** [comment]
@seanh good point - FWIW, I did not come up with this solution for the user - this was what they asked for in the ticket. In general I try not to propose solutions, but I do think it's important to surface the actual content of feature requests that come from users.

**Comment by klemay** [comment]
Linking #528
